bpf/lib/policy.h

```diff
@@ -23,11 +23,21 @@
 #include "maps.h"
 
 /**
- * MinimalNumericIdentity describes the lowest possible identity
- * allocated to endpoints. Numbers lower than this indicated
- * reserved identities.
+ * NUMERIC_IDENTITY_MASK is used to determine whether an identity is one of the
+ * reserved identities that are not handed out to endpoints.
+ *
+ * Specifically, if the (identity & mask == 0) then the identity is one of the
+ * below:
+ * - ID_UNKNOWN (0)
+ * - ID_HOST    (1)
+ * - ID_WORLD   (2)
+ * - ID_CLUSTER (3)
+ *
+ * It does not include:
+ * - ID_HEALTH (4)
+ * - ID_INIT   (5)
  */
-#define MINIMAL_NUMERIC_IDENTITY 256
+#define NUMERIC_IDENTITY_MASK 0xFFFFFFFC
```

Comment:
Why not change the `MINIMAL_NUMERIC_IDENTITY` to a new variable and the value be 3?

Reply:
I'm sure I had some reason when I wrote this patch, but right now I can't figure out what it was. This suggestion is much simpler, so I'll change it to do this.